### ISSUE TITLE
RtNetwork.java:63-67: Implement pull method. Pulling a...

### ISSUE THREAD

**Issue by 0pdd** [zold-io/java-api#41]
The puzzle `5-dec07794` from #5 has to be resolved:

https://github.com/zold-io/java-api/blob/e7bf16247075ec2bb23081b61e1835fa44551621/src/main/java/io/zold/api/RtNetwork.java#L63-L67

The puzzle was created by Paulo Lobo on 19-Jul-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by victornoel** [comment]
@0crat waiting for #49 to be reviewed and merged.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@6572b8cd-01c4-44a8-8ef7-8ac6377e8d40, thread: pool-40-thread-1
com.zerocracy.pm.Claims[109] java.lang.IllegalStateException: Duplicate claims are not allowed in [CAZUREFND](https://www.0crat.com/p/CAZUREFND), can't add this XML to 3 existing ones (Add job to WBS;{job=gh:zold-io/java-api#49, role=REV};github;zold-io/java-api;49,Add job to WBS;{job=gh:zold-io/java-api#49, role=REV};github;zold-io/java-api;49,Register impediment;{comment=406871044, job=gh:zold-io/java-api#41, issue=41, repo=zold-io/java-api};github;zold-io/java-api;41;406871044):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2031433426">
   <created>2018-07-22T14:33:22.249Z</created>
   <type>Register impediment</type>
   <token>github;zold-io/java-api;41;406871044</token>
   <author>victornoel</author>
   <params>
      <param name="job">gh:zold-io/java-api#41</param>
      <param name="repo">zold-io/java-api</param>
      <param name="issue">41</param>
      <param name="comment">406871044</param>
   </params>
</claim>

```

[0.26.4](https://github.com/zerocracy/farm): Issue: zold-io/java-api#41, Comment: 406871044

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@6572b8cd-01c4-44a8-8ef7-8ac6377e8d40, thread: pool-40-thread-1
com.zerocracy.radars.github.ReSafe[81] java.io.IOException: java.lang.IllegalStateException: Duplicate claims are not allowed in [CAZUREFND](https://www.0crat.com/p/CAZUREFND), can't add this XML to 3 existing ones (Add job to WBS;{job=gh:zold-io/java-api#49, role=REV};github;zold-io/java-api;49,Add job to WBS;{job=gh:zold-io/java-api#49, role=REV};github;zold-io/java-api;49,Register impediment;{comment=406871044, job=gh:zold-io/java-api#41, issue=41, repo=zold-io/java-api};github;zold-io/java-api;41;406871044):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2031433426">
   <created>2018-07-22T14:33:22.249Z</created>
   <type>Register impediment</type>
   <token>github;zold-io/java-api;41;406871044</token>
   <author>victornoel</author>
   <params>
      <param name="job">gh:zold-io/java-api#41</param>
      <param name="repo">zold-io/java-api</param>
      <param name="issue">41</param>
      <param name="comment">406871044</param>
   </params>
</claim>

com.zerocracy.pm.Claims[109] java.lang.IllegalStateException: Duplicate claims are not allowed in [CAZUREFND](https://www.0crat.com/p/CAZUREFND), can't add this XML to 3 existing ones (Add job to WBS;{job=gh:zold-io/java-api#49, role=REV};github;zold-io/java-api;49,Add job to WBS;{job=gh:zold-io/java-api#49, role=REV};github;zold-io/java-api;49,Register impediment;{comment=406871044, job=gh:zold-io/java-api#41, issue=41, repo=zold-io/java-api};github;zold-io/java-api;41;406871044):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2031433426">
   <created>2018-07-22T14:33:22.249Z</created>
   <type>Register impediment</type>
   <token>github;zold-io/java-api;41;406871044</token>
   <author>victornoel</author>
   <params>
      <param name="job">gh:zold-io/java-api#41</param>
      <param name="repo">zold-io/java-api</param>
      <param name="issue">41</param>
      <param name="comment">406871044</param>
   </params>
</claim>

```

[0.26.4](https://github.com/zerocracy/farm): Issue: zold-io/java-api#41, Action: `created`

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat waiting for #49 to be reviewed and merged. ([here](https://github.com/zold-io/java-api/issues/41#issuecomment-406871044))

@victornoel The impediment for [#41](https://github.com/zold-io/java-api/issues/41) was registered successfully by @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zold-io/java-api/issues/41#issuecomment-406871201))

@victornoel This is what I know about this job in [CAZUREFND](https://www.0crat.com/p/CAZUREFND), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#41](https://github.com/zold-io/java-api/issues/41) is [in scope](http://www.zerocracy.com/policy.html#14) for 3days
  * The role is `DEV`
  * The job is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel) for [19hr](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @victornoel[/z](https://www.0crat.com/u/victornoel) will get 50% of their hourly rate for this job
  * The job has an [impediment](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * Job [footprint](https://www.0crat.com/footprint/CAZUREFND?q=%7Bjob%3A%27gh%3Azold-io%2Fjava-api%2341%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0pdd** [comment]
The puzzle `5-dec07794` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#56](https://github.com/zold-io/java-api/issues/56) is still not solved.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#41](https://github.com/zold-io/java-api/issues/41) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#79](https://github.com/zold-io/java-api/issues/79), [#80](https://github.com/zold-io/java-api/issues/80) are still not solved; solved: [#56](https://github.com/zold-io/java-api/issues/56).
